core/controllers/learner_group.py

```diff
@@ -827,3 +958,135 @@ def get(self, username):
                 user_id, story_ids))
 
         self.render_json(stories_chapters_progress)
+
+
+class LearnerDashboardLearnerGroupsHandler(base.BaseHandler):
+    """Handles fetching of learners groups on learner dashboard."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
```

Comment:
Missing types:

For `URL_PATH_ARGS_SCHEMAS` Use Dict[str, str]
For `HANDLER_ARGS_SCHEMAS` use Dict[str, Dict[str, str]]

Reply:
Done here and everywhere else